src/csharp/Extensions/Microsoft.Spark.Extensions.Delta.E2ETest/SparkCollection.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace Microsoft.Spark.Extensions.Delta.E2ETest
+{
+    [CollectionDefinition(Constants.DeltaTestContainerName)]
+    public class SparkCollection : ICollectionFixture<DeltaFixture>
```

Comment:
`DeltaCollection` or `DeltaTestsCollection`? Also, wouldn't it be easier to see if this is defined in the fixture file (same as how Spark.E2Etest is set up.

Reply:
Changed